numpy/core/tests/test_cpu_features.py

```diff
@@ -103,6 +115,201 @@ def load_flags_auxv(self):
                     hwcap_value[1].upper().decode().split()
                 )
 
+class TestEnvPrivation:
+    cwd = pathlib.Path(__file__).parent.resolve()
+    env = os.environ.copy()
+    _enable = os.environ.pop('NPY_ENABLE_CPU_FEATURES', None)
+    _disable = os.environ.pop('NPY_DISABLE_CPU_FEATURES', None)
+    SUBPROCESS_ARGS = dict(cwd=cwd, capture_output=True, text=True, check=True)
+    unavailable_feats = [
+        feat for feat in __cpu_dispatch__ if not __cpu_features__[feat]
+    ]
+    UNAVAILABLE_FEAT = (
+        None if len(unavailable_feats) == 0
+        else unavailable_feats[0]
+    )
+    BASELINE_FEAT = None if len(__cpu_baseline__) == 0 else __cpu_baseline__[0]
+    SCRIPT = """
+def main():
```

Comment:
bad merge? This code is out of place, which means the tests below do not run.

Reply:
I'm not sure what you mean -- the tests run for me, unless I'm misunderstanding

Reply:
Something is off. Coverage claims that most of the new tests are not running.

Reply:
Ah, but you mentioning this makes me think that coverage was broken by Ralf's PR to not run most jobs on merge.  The coverage needs to run on merges to be able to do comparisons.